v1/card/card.gunk

```diff
@@ -165,11 +287,49 @@ type CardService interface {
 	// }
 	GetCard(GetCardRequest) Card
 
+	// GetCardForCurrentUser retrieves the detail of a cards information for the specific user, selected by user_id.
+	//
+	// +gunk http.Match{
+	//         Method: "GET",
+	//         Path:   "/v1/card/user/{UserID}",
+	// }
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"Card"},
+	//         Description: "Retrieves all available cards for specific user, selected by the user_id",
+	//         Summary:     "Retrieves all available cards",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "200": openapiv2.Response{
+	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/cardGetCardForSpecificUser",
```

Comment:
is it GetCardForSpecificUser or GetCardForCurrentUser?

Reply:
You can define the name as `GetUserCards` or `GetCardsByUserID`